sensor/upgrader/upgradectx/context.go

```diff
@@ -345,3 +352,8 @@ func (c *UpgradeContext) ListCurrentObjects() ([]*unstructured.Unstructured, err
 
 	return objects, nil
 }
+
+// IsPodSecurityEnabled returns whether or not pod security polices are enabled for this cluster
+func (c *UpgradeContext) IsPodSecurityEnabled() bool {
+	return c.podSecurityPoliciesSupported
```

Comment:
This field seems to be [set](https://github.com/stackrox/stackrox/pull/6505/files#diff-acc1578eababf29f1daa395679380b56b089fa2d0b0fe30ee316ffce9917bb1fR110) based on `OrderedBundleResourceTypes` which according to the comment will contain PSPs for all eternity, so I cannot see how this function could ever return anything else than `true`?